### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by rachelcossich**

### CODE REVIEW

File: tests/login/TestLogin.robot [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Documentation    Test suite for serverest API - /login endpoint
+Test Tags        @login
+
+*** Test Cases ***
+Login successfully
+    [Tags]    @test     @regression
+    Login    Fulano da Silva        teste
+    Should Be Equal As Numbers    ${response.status_code}            200
+    Should Be Equal As Strings    ${response.json()['message']}       Login realizado com sucesso!
+
```
Comment (uLucasFraga) [inline tests/login/TestLogin.robot]: remove this line, please

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+robotframework==7.0
+robotframework-requests==0.9.6
+robotframework-faker==5.0.0
+
```
Comment (uLucasFraga) [inline requirements.txt]: remove this line, please

**Review by uLucasFraga — CHANGES_REQUESTED** [review]

File: tests/login/TestLogin.robot (hunk 1 above)
Comment (rachelcossich) [reply tests/login/TestLogin.robot]: fix

File: requirements.txt (hunk 2 above)
Comment (rachelcossich) [reply requirements.txt]: fix

**Review by uLucasFraga — APPROVED** [review]

**Comment by uLucasFraga** [comment]
LGTM!

**Pull request merged**